include/librealsense2/h/rs_internal.h

```diff
@@ -49,6 +48,23 @@ rs2_context* rs2_create_recording_context(int api_version, const char* filename,
  */
 rs2_context* rs2_create_mock_context(int api_version, const char* filename, const char* section, rs2_error** error);
 
+/**
+ TODO
+**/
+rs2_device* rs2_create_bypass_device(rs2_error** error);
+void rs2_bypass_add_sensor(rs2_device* dev, const char* sensor_name, rs2_error** error);
```

Comment:
I think it would make sense to have some correlation between add_sensor and on_frame with regards to sensor identification- here we pass a string, but in on_frame we pass an int (and the device does not provide indices nor does the sensor know its own index).